frontend/common/src/components/UserProfile/ExperienceByTypeListing.tsx

```diff
@@ -93,8 +93,10 @@ const ExperienceByTypeListing: React.FunctionComponent<
       {personalExperiences.length > 0 ? (
         <ExperienceByType
           title={intl.formatMessage({
-            defaultMessage: "Personal",
-            id: "NDx+B0",
```

Comment:
I suppose to go the extra mile you could delete these from Fr.json 🤔 
Unless there is a reason to keep them around (potentially need the translated words)

Reply:
Ya, I thought about this too (and have before https://github.com/GCTC-NTGC/gc-digital-talent/issues/4489). The weird part about these strings is that one of them, Education `jtygmI`, is used elsewhere.

Reply:
ah just one of them, interesting, better safe than sorry